src/utils.ts

```diff
@@ -157,6 +156,87 @@ export const updatePrDetails = async (client: github.GitHub, prData: PullsUpdate
   }
 };
 
+/**
+ *  Add a comment to a PR
+ * @param {github.GitHub} client
+ * @param {IssuesCreateCommentParams} comment
+ */
+export const addComment = async (client: github.GitHub, comment: IssuesCreateCommentParams) => {
+  try {
+    await client.issues.createComment(comment);
+  } catch (error) {
+    core.setFailed(error.message);
+    process.exit(1);
+  }
+};
+
+/**
+ *  Get a comment based on story title and PR title similarity
+ * @param {string} storyTitle
+ * @param {string} prTitle
+ * @returns {string}
+ */
+export const getCommentBody = (storyTitle: string, prTitle: string): string => {
+  const matchRange: number = similarity.compareTwoStrings(storyTitle, prTitle);
+  if (matchRange < 0.4) {
+    return `<p>
+    Knock Knock! 🔍
+  </p>
+
+  <p>
+    Just thought I'd let you know that your <em>PR title</em> and <em>story title</em> look <strong>quite different</strong>. PR titles
+    that closely resemble the story title make it easier for reviewers to understand the context of the PR.
+  </p>
+
+  <blockquote>
+    An easy-to-understand PR title a day makes the reviewer review away! 😛⚡️
+  </blockquote>
+
+  <table>
+    <tr>
+      <th>Story Title</th>
+      <td>${storyTitle}</td>
+    </tr>
+    <tr>
+        <th>PR Title</th>
+        <td>${prTitle}</td>
+      </tr>
+  </table>
+
+  <p>
+    Check out this <a href="https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests">guide</a> to learn more about PR best-practices.
+  </p>
+  `;
+  } else if (matchRange >= 0.4 && matchRange <= 0.75) {
+    return `<p>
+    Let's make that PR title a 💯 shall we? 💪
+    </p>
+
+    <p>
+    Your <em>PR title</em> and <em>story title</em> look <strong>slightly different</strong>. Just checking in to know if it was intentional!
+    </p>
+
+    <table>
+      <tr>
+        <th>Story Title</th>
+        <td>${storyTitle}</td>
+      </tr>
+      <tr>
+          <th>PR Title</th>
+          <td>${prTitle}</td>
+        </tr>
+    </table>
+
+    <p>
+      Check out this <a href="https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests">guide</a> to learn more about PR best-practices.
+    </p>
+    `;
+  }
+  return `<p>I'm a bot and I 👍 this PR title. 🤖</p>
+
+  <img src="https://media.giphy.com/media/XreQmk7ETCak0/giphy.gif" width="400" />`;
```

Comment:
giphy might remove this image. Should we use it from cleartax cdn?

Reply:
They don't usually do that. It's meant to be shared.

Since `pr-lint` is public, let's not use our CDN for everyone else's usage :P